beacon-chain/sync/initial-sync/service.go

```diff
@@ -44,8 +46,9 @@ type Config struct {
 // CrystallizedStateBufferSize determines the buffer size of thhe `crystallizedStateBuf` channel.
 func DefaultConfig() Config {
 	return Config{
-		SyncPollingInterval:         1 * time.Second,
+		SyncPollingInterval:         time.Duration(int64(params.BeaconConfig().SlotDuration)) * time.Second * 4,
```

Comment:
What is this constant 4? Can we predefined this constant?

Reply:
Fixed into a config var